src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -232,6 +232,9 @@ def load_arguments(self, _):
     with self.argument_context('functionapp list-flexconsumption-locations') as c:
         c.argument('zone_redundant', arg_type=get_three_state_flag(),
                    help='Filter the list to return only locations which support zone redundancy.', is_preview=True)
+        c.argument('details', arg_type=get_three_state_flag(),
```

Comment:
Can you rename this to `show-details`? I think just `details` is a bit confusing.